src/corehost/common/pal.h

```diff
@@ -54,12 +54,15 @@
 #if defined(_WIN32)
 #define LIB_PREFIX
 #define MAKE_LIBNAME(NAME) (_X(NAME) _X(".dll"))
+#define BASE_HOST_RID _X("win")
```

Comment:
> Thinking a bit more, what happens if OSX 10.13 (or win11) is incompatible with its predecessors and breaks previous assets?

If that was the case, we would break the chain of the RID fallback graph.  We would say that `osx.10.13` maps straight to `osx`, skipping its predecessors.  This is what happens for `ubuntu`.  `ubuntu16` => `ubuntu`, skipping `ubuntu14`.